Makefile.PL

```diff
@@ -20,6 +20,8 @@ WriteMakefile(
         'HTML::Table::FromDatabase' => 1.10,
         'CGI::FormBuilder' => 3.10,
         'DBD::SQLite'   => 1.50,
+        'DBD::mysql' => 0,
```

Comment:
not sure we should make DBD::mysql a hard requirement if it's only used in an example